codalab/worker/worker_run_state.py

```diff
@@ -645,8 +738,20 @@ def _transition_from_FINALIZING(self, run_state):
         move on to the FINISHED state. Can also remove bundle_path now.
         """
         if run_state.is_restaged:
+            log_bundle_transition(
+                run_state.bundle.uuid,
+                run_state.stage,
+                RunStage.RESTAGED,
+                'the bundle is going to be sent back to the server',
+            )
             return run_state._replace(stage=RunStage.RESTAGED)
         elif run_state.finalized:
+            log_bundle_transition(
+                run_state.bundle.uuid,
+                run_state.stage,
+                RunStage.FINISHED,
+                'a full worker cycle has passed',
```

Comment:
Do we need a reason here?

Reply:
a brief explanation from comments - https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/worker_run_state.py#L643

Reply:
Can you include it in the logs?